src/libraries/Microsoft.PowerFx.Core/Utils/TextFormatUtils.cs

```diff
@@ -2,15 +2,85 @@
 // Licensed under the MIT license.
 
 using System;
+using System.Collections.Immutable;
+using System.Globalization;
 using System.Text.RegularExpressions;
+using System.Xml;
 
 namespace Microsoft.PowerFx.Core.Utils
 {
+    internal class TextFormatArgs
+    {
+        internal string FormatCultureName { get; set; }
+
+        internal string FormatArg { get; set; }
+
+        internal bool HasDateTimeFmt { get; set; }
+
+        internal bool HasNumericFmt { get; set; }
+    }
+
     internal sealed class TextFormatUtils
```

Comment:
>TextFormatUtils [](http://example.com/codeflow?start=26&length=15)

We should be sure our tests give 100% code coverage on this file.

Reply:
run code coverage and got all lines cover